ArjunaJTA/jta/classes/com/arjuna/ats/internal/jta/transaction/arjunacore/subordinate/SubordinateAtomicAction.java

```diff
@@ -55,6 +55,7 @@ public SubordinateAtomicAction ()
 	public SubordinateAtomicAction (int timeout)
 	{
 		super();
+		dynamic1PC = false;
```

Comment:
Because ServerTransaction is a subordinateTransaction that can have doPrepare called on it the same as SubordinateAtomicAction